<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertNode</class>
 <widget class="QDialog" name="InsertNode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>427</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="labName">
       <property name="text">
        <string>&amp;Name</string>
       </property>
       <property name="buddy">
        <cstring>lineEditName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labDescription">
       <property name="text">
        <string>D&amp;escription</string>
       </property>
       <property name="buddy">
        <cstring>lineEditDescription</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditDescription"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labProperty">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Property</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="buddy">
        <cstring>rBtnDICD</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="chkBoxPlaceholer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Placeholer</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="rBtnDICD">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;DICD</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="rBtnDDCI">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>DDC&amp;I</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labNote">
       <property name="text">
        <string>No&amp;te</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="buddy">
        <cstring>plainTextEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="pBtnHelp">
       <property name="text">
        <string>&amp;Help</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pBtnCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pBtnOk">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditDescription</tabstop>
  <tabstop>pBtnOk</tabstop>
  <tabstop>pBtnCancel</tabstop>
  <tabstop>pBtnHelp</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>rBtnDICD</tabstop>
  <tabstop>rBtnDDCI</tabstop>
  <tabstop>chkBoxPlaceholer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
